docs/workflows/aws.md

```diff
@@ -13,44 +13,62 @@ The **RTC_S1** pipeline must be run for every new scene acquired by Sentinel-1.
 
 The static layers can be re-used across a given burst_id, saving the storage required if they were created with each acquisition. For example, every 12 days sentinel 1A will capture the burst `t070_149815_iw3`. The same `local_incidence_angles.tif` can be used for each repeat pass, only the dialectic properties of the surface will change over time. The static layer is therefore referenced in a given **RTC_S1** product STAC metadata.
 
-TODO: Ensure this is how products are structured 
+To link a **RTC_S1** product to the appropriate  **RTC_S1_STATIC** layers, the **RTC_S1_STATIC** layer product must exist for all bursts over the scene of interest. The linkage is then specified when the workflow is run to create **RTC_S1** products. See the example *Make static layers (RTC_S1_STATIC) for a burst and link it to a backscatter product (RTC_S1)* below.
```

Comment:
Might be worth having an extra line clarifying that linking doesn't link products in the processing sense.

Reply:
I agree -- alternatively/additionally, you could rephrase to say that the linking occurs in the metadata. You could then add that the reason we do this is to save storage space.